modules/ipi-install-network-requirements.adoc

```diff
@@ -9,6 +9,12 @@ Installer-provisioned installation of {product-title} involves several network r
 
 image::210_OpenShift_Baremetal_IPI_Deployment_updates_0122_2.png[Installer-provisioned networking]
 
+
+[id="network-requirements-increase-mtu_{context}"]
+== Increase the network MTU
+
+Before deploying {product-title}, increase the network maximum transmission unit (MTU) to 1500 or more. If the MTU is lower than 1500, the Ironic image used to boot the node might fail to communicate with the Ironic inspector pod and inspection will fail. If this occurs, installation will stop, because the nodes will not become available for installation. 
```

Comment:
I think you can simplify the tenses here without impacting the meaning.

```suggestion
Before deploying {product-title}, increase the network maximum transmission unit (MTU) to 1500 or more. If the MTU is lower than 1500, the Ironic image that is used to boot the node might fail to communicate with the Ironic inspector pod, and inspection will fail. If this occurs, installation stops because the nodes are not available for installation. 
```

Reply:
@kalexand-rh Accepted the suggestion and squashed the commit count.